### DISCUSSION TITLE
2023/06/使用 Mastodon 作为博客的评论系统/

### DISCUSSION THREAD

**Discussion by SouthFox-D** [SouthFox-D/SouthFox-D.github.io#85] [博客评论]
[https://blog.southfox.me/2023/06/使用 Mastodon 作为博客的评论系统/](https://blog.southfox.me/2023/06/%E4%BD%BF%E7%94%A8%20Mastodon%20%E4%BD%9C%E4%B8%BA%E5%8D%9A%E5%AE%A2%E7%9A%84%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F/)

**Comment by tongtong0815** [comment]
评论（
